code/src/components/TodoSummary.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import { useSelector } from 'react-redux'
+import styled from 'styled-components/macro'
+
+export const TodoSummary = () => {
+  const list = useSelector(store => store.todos.list)
```

Comment:
Same here, if you only need the items-array, get only that:
`const items = useSelector(store => store.todos.list.items)`